packages/enhanced/src/lib/container/ContainerEntryModule.ts

```diff
@@ -4,16 +4,27 @@
 */
 
 'use strict';
-import AsyncDependenciesBlock from 'webpack/lib/AsyncDependenciesBlock';
-import Dependency from 'webpack/lib/Dependency';
-import Template from 'webpack/lib/Template';
-import Module from 'webpack/lib/Module';
-import * as RuntimeGlobals from 'webpack/lib/RuntimeGlobals';
-import { OriginalSource, RawSource } from 'webpack-sources';
-import { JAVASCRIPT_MODULE_TYPE_DYNAMIC } from 'webpack/lib/ModuleTypeConstants';
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+import type { Dependency, Compilation } from 'webpack';
+
+const makeSerializable = require(
+  normalizeWebpackPath('webpack/lib/util/makeSerializable'),
+) as typeof import('webpack/lib/util/makeSerializable');
+const { sources: webpackSources } = require(
+  normalizeWebpackPath('webpack'),
+) as typeof import('webpack');
+const { AsyncDependenciesBlock, Template, Module, RuntimeGlobals } = require(
+  normalizeWebpackPath('webpack'),
+) as typeof import('webpack');
+const { JAVASCRIPT_MODULE_TYPE_DYNAMIC } = require(
+  normalizeWebpackPath('webpack/lib/ModuleTypeConstants'),
+) as typeof import('webpack/lib/ModuleTypeConstants');
+const StaticExportsDependency = require(
+  normalizeWebpackPath('webpack/lib/dependencies/StaticExportsDependency'),
+) as typeof import('webpack/lib/dependencies/StaticExportsDependency');
+
 import ContainerExposedDependency from './ContainerExposedDependency';
-import StaticExportsDependency from 'webpack/lib/dependencies/StaticExportsDependency';
-import type Compilation from 'webpack/lib/Compilation';
+// eslint-disable-next-line @typescript-eslint/no-restricted-imports
```

Comment:
you can configure eslint to allow type imports, not not allow normal imports. so if it has import type, you can make eslint ignore those. @2heal1

Reply:
yes, done